Info.plist

```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIdentifier</key>
-	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<string>org.swift.XCTest</string>
```

Comment:
Could you set this as the value for the `PRODUCT_BUNDLE_IDENTIFIER` setting in the project instead of putting it directly in the `Info.plist`?